cmd/podman/umount.go

```diff
@@ -16,13 +16,23 @@ var (
 			Name:  "all, a",
 			Usage: "umount all of the currently mounted containers",
 		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "force the complete umount all of the currently mounted containers",
```

Comment:
I'm not sure I grok this. It looks like you copypasted from --all but didn't finish writing the description?